pkg/crc/machine/machine.go

```diff
@@ -107,7 +107,7 @@ func createLibMachineClient(debug bool) (*libmachine.Client, func(), error) {
 	}, nil
 }
 
-func Start(startConfig StartConfig) (StartResult, error) {
+func (client *LibMachineClient) Start(startConfig StartConfig) (StartResult, error) {
 	var crcBundleMetadata *bundle.CrcBundleInfo
 
 	libMachineAPIClient, cleanup, err := createLibMachineClient(startConfig.Debug)
```

Comment:
Seeing this, I don't think the `client *LibMachineClient` naming is quite right, as even if we have a `LibMachineClient` already, we need to call a method named `createLibMachineClient` to get a different kind of LibMachineClient?
`client *MachineClient` is probably good enough as a name, as it's a client used in the `machine` package?

Reply:
When I changed to MachineClient the linter complains:

```
pkg/crc/machine/client.go:14:6: type name will be used as machine.MachineClient by other packages, and that stutters; consider calling this Client (golint)
type MachineClient struct{}
``` 

Other idea ? It's hard to find  a good name here :(

Reply:
I chose `client` for the moment (and the interface is `Client`). Not sure if it's better but it's a start.

Reply:
Yup, only remembered the linter would complain while away from the computer :) I guess if we try to call it `Machine` the linter will also fail? `Instance` might do, but not much better than `Client`. Or `Manager` as this really is about managing our crc machine(s)?